### DISCUSSION TITLE
Amethyst 0.15.6 Beta

### DISCUSSION THREAD

**Discussion by ianyh** [ianyh/Amethyst#1204] [Announcements]
* Fixes for throwing windows between spaces on macOS 12.2+. Thanks, @dzanotto! #1184, #1174, #1203, #1192
* Add regular expression matching for window title-based floating. Thanks, @SebastienWae! #1166, #981 

<hr /><em>This discussion was created from the release <a href='https://github.com/ianyh/Amethyst/releases/tag/v0.15.6b'>Amethyst 0.15.6 Beta</a>.</em>

**Comment by ianyh** [comment]
This hasn't gone out as an update yet, but you can download it now if you need it.

**Comment by ianyh** [comment]
Okay, put it out on the dev channel.

**Comment by brush701** [comment]
FYI this doesn't seem to fix #1184 for me. Same behavior as before the release running Monterey 12.2.1.

**Reply by mwz** [reply]
It does work with the setting to "Follow thrown windows between spaces" disabled.

**Reply by brush701** [reply]
Doesn't seem to make a difference for me unfortunately.

**Reply by ianyh** [reply]
Could you open a new bug with details?
